docs/cli.md

```diff
@@ -146,6 +146,12 @@ You could run the CLI as follows:
     ...
     ```
 
+## Upgrading to a newer version of the CLI
+New releases of the Neurobagel CLI will occasionally introduce breaking changes to the data model for subject-level information in a `.jsonld` graph file.
```

Comment:
```suggestion
Neurobagel is under active, early development and future releases of the CLI may introduce breaking changes to the data model for subject-level information in a `.jsonld` graph file. Breaking changes will be highlighted in the release notes!
```
Something like this to make it clear that we aren't just going to randomly break stuff.